usecases/schema/migrate/fs/file_structure_migration.go

```diff
@@ -114,7 +114,7 @@ func assembleFSMigrationPlan(entries []os.DirEntry, rootPath string, fm *fileMat
 			for _, cs := range css {
 				plan.append(cs.class, cs.shard,
 					path.Join(strings.ToLower(entry.Name()), cs.shard, "compressed_objects"),
-					path.Join("lsm", helpers.VectorsCompressedBucketLSM))
```

Comment:
@antas-marcin, could you please take a look on this line. I think it is wrong. We might need the name of the vectors here too. Although I do not know if we are migrating here directly to the new version or to an old version (I think it is the former, hence the line is wrong)